data_access/eth_chain.py

```diff
@@ -1047,25 +1044,21 @@ def __init__(self, event_client_type, address=""):
             self._contract_addresses = [BEANSTALK_ADDR, FERTILIZER_ADDR]
             self._events_dict = BEANSTALK_EVENT_MAP
             self._signature_list = BEANSTALK_SIGNATURES_LIST
-            self._set_filters()
         elif self._event_client_type == EventClientType.MARKET:
             self._contracts = [get_beanstalk_contract(self._web3)]
             self._contract_addresses = [BEANSTALK_ADDR]
             self._events_dict = MARKET_EVENT_MAP
             self._signature_list = MARKET_SIGNATURES_LIST
-            self._set_filters()
         elif self._event_client_type == EventClientType.BARN_RAISE:
             self._contracts = [get_fertilizer_contract(self._web3)]
             self._contract_addresses = [FERTILIZER_ADDR]
             self._events_dict = FERTILIZER_EVENT_MAP
             self._signature_list = FERTILIZER_SIGNATURES_LIST
-            self._set_filters()
```

Comment:
why did you remove these calls to reset the filters? these were put in for edge cases where the filters change during runtime.

Reply:
It was moved to the end of the if/else ladder instead of including in each condition